src/send_kcidb.py

```diff
@@ -175,15 +176,16 @@ def _parse_build_node(self, origin, node):
             'architecture': node['data'].get('arch'),
             'compiler': node['data'].get('compiler'),
             'config_name': node['data'].get('defconfig'),
-            'valid': node['result'] == 'pass',
+            'valid': node['result'] if node['result'] in ('pass', 'fail') else None,
```

Comment:
Previously `valid` contained a bool (`True`/`False`) and now it might be str ("pass"/"fail") or None - was that expected? Is it handled on the KCIDB side?

Reply:
You're right. Thanks a lot for catching this.
I'll fix it.